### PR TITLE
Mj 3849 gsb spacing

### PR DESCRIPTION
**Pull request by micahjones13**
## Brief Description

Spacing updates for GSB and it's close counterparts, monitoring icon and app-state.

## JIRA Link

GSB -> https://rocketcom.atlassian.net/browse/ASTRO-3849
App state -> https://rocketcom.atlassian.net/browse/ASTRO-3843
Monitoring icon -> https://rocketcom.atlassian.net/browse/ASTRO-3850

## General Notes
I tackled monitoring icon with GSB since it's a very common use case, and needed it for Figma VRT's anyway.
App state is a separate component in Figma, but it part of GSB in code - hence why it's here too. 

## Motivation and Context

Spacing effort 👾 

## Issues and Limitations

Monitoring icon is a bit off still due to the status symbol mismatches in size - this will be addressed at a later stage when we swap in SVG's for the status symbols. 
The only thing not lining up in GSB is the text due to a font difference. It's like half a px off. 
Clock does not look the same inside GSB as it does in Figma, due to a couple things:
- Clocks own spacing 
- It's a default slot in code, which means not a ton of specific styling since anything could go there. External devs can style it however they see fit. 

## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Checklist

- [ ] This PR adds or removes a Storybook story.
- [ ] I have added tests to cover my changes.
- [ ] Regressions are passing and/or failures are documented
- [ ] Changes have been checked in evergreen browsers

### CODE REVIEW

File: packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss [hunk 1]
```diff
@@ -6,11 +6,12 @@
     z-index: 50;
     top: 0;
     left: 0;
-    height: 100px;
+    height: var(--spacing-20); //80 px
     width: -webkit-fill-available;
     width: -moz-available;
     width: stretch;
-    padding: 0 1.875rem;
+    // padding: 0 1.875rem;
```
Comment (FMorrison87) [inline packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss]: Remove commented code?

File: packages/web-components/src/components/rux-global-status-bar/test/index.html [hunk 2]
```diff
@@ -17,8 +17,76 @@
         <script nomodule src="/build/astro-web-components.js"></script>
 
         <link rel="stylesheet" href="/build/astro-web-components.css" />
+        <script
+            type="module"
```
Comment (FMorrison87) [inline packages/web-components/src/components/rux-global-status-bar/test/index.html:21]: Mark had me comment out the script and holster. Apparently it affects Cypress.

Comment (micahjones13) [reply packages/web-components/src/components/rux-global-status-bar/test/index.html:21]: Thank you! I've been punching cypress in the face but it wasn't helping.

File: packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss [hunk 3]
```diff
@@ -63,30 +71,32 @@ slot[name='left-side'] > .shifted-up,
...
-    margin-bottom: 0;
-    margin-top: 0;
-    height: 32px;
+    margin-bottom: var(--spacing-0);
+    margin-top: var(--spacing-0);
     font-family: var(--font-heading-1-font-family);
     font-size: var(--font-heading-1-font-size);
     font-weight: var(--font-heading-1-font-weight);
     letter-spacing: var(--font-heading-1-letter-spacing);
+    line-height: var(--line-height-2xl);
```
Comment (markacianfrani) [inline packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss:86]: is there a specific reason for --line-height-2xl over --font-heading-1-line-height?

Comment (micahjones13) [reply packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss:86]: Nope, just an oversight. Good catch!

**Review by kiley-mitti — APPROVED** [review]
I didn't realize there were spacing tokens between 0 and 1. I went back and refactored some of my own code after checking this out. :)

**Review by markacianfrani — APPROVED** [review]

File: packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss [hunk 4]
```diff
@@ -112,10 +122,9 @@ slot[name='left-side'] > .shifted-up,
     font-size: var(--font-body-2-font-size);
     font-weight: var(--font-body-2-font-weight);
     letter-spacing: var(--font-body-2-letter-spacing);
-    padding: 2px 6px;
+    padding: calc(var(--spacing-050) + var(--spacing-025)) var(--spacing-2); //3px 8px
     text-align: center;
-    height: 22px;
-    margin-right: 6px;
+    margin-right: var(--spacing-2); //6px
```
Comment (jonathantneal) [inline packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss:127]: **NIT**: When the root font size is `16px`, the `--spacing-2` variable ends up resolving to `8px`. The `6px` note here may be confusing.

**Pull request merged**